tests/TestCase.php

```diff
@@ -46,15 +52,14 @@ public function createApplication()
     {
         $app = require __DIR__.'/../bootstrap/app.php';
 
-        $app->make(Illuminate\Contracts\Console\Kernel::class)->bootstrap();
+        $app->make(Kernel::class)->bootstrap();
+
+        $this->getTestFiles();
 
         return $app;
     }
 
-    /**
-     * @before
-     */
-    public function initializeTestFiles()
+    public function getTestFiles()
```

Comment:
Just incase this is questioned. PHP Unit 6 ends up calling this prior to the application being booted, which results in no `Storage` facade having been registered - and all tests failing 😢 .

Reply:
👍

Reply:
Just realised I probably could have just removed the docblock without renaming, but 🤷‍♀️ it's done now. :P